packages/perseus/src/components/__tests__/zoomable.test.tsx

```diff
@@ -23,11 +22,29 @@ const mockSize = (
     }
 };
 
+// The zoomable does some measuring after the initial render to determine what
```

Comment:
As I dug into these tests for the Zoomable, I discovered that they weren't really validating things correctly. I've fixed them up so that I _think_ they are now correct _and_ compatible with user-event@14